cfgov/unprocessed/apps/owning-a-home/js/explore-rates/params.js

```diff
@@ -0,0 +1,69 @@
+const domValues = require( './dom-values' );
+
+let UNDEFINED;
+
+// List all the parameters the user can change and set
+// their default values.
+// `verbotenKeys` are as follows:
+// 9 = tab
```

Comment:
Should these be constants like we've done elsewhere (specifically thinking of the multi-select/typeahead thing for filtering options).

Reply:
The lookup for these uses a numeric value and there's another group of keys that's a bigger group than this, so probably best to leave as is till we're certain this is stable and then refactor.